.devcontainer/Dockerfile

```diff
@@ -0,0 +1,123 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
Is this `.devcontainer` useful for all contributors to the project? or is it something specific for your local testing/environment that should be included in the `.gitignore` file?